src/main/java/org/spongepowered/api/attributes/Attributes.java

```diff
@@ -0,0 +1,15 @@
+package org.spongepowered.api.attributes;
+
+public class Attributes {
```

Comment:
You also might want to add to `GameRegistry` to retrieve `Optional<Attribute> getAttributeByName(String)` and `List<Attribute> getAvailableAttributes()` or something similar to what is available in the registry.